ERCS/erc-7208.md

```diff
@@ -0,0 +1,598 @@
+---
+eip: 7208
+title: On-Chain Data Container
+description: Abstracting logic away from storage for RWA Tokenization
+author: Rachid Ajaja <rachid@allianceblock.io>, Alexandros Athanasopulos (@Xaleee), Pavel Rubin (@pash7ka), Sebastian Galimberti Romano (@galimba)
+discussions-to: https://ethereum-magicians.org/t/erc-7208-on-chain-data-container/14778
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-09
+requires: 721
+---
+
+## Abstract
+
```

Comment:
I think you can move some of your abstract into motivation (parts that deal with "why this proposal exists"), and add a bit more technical detail on what an ODC _is_. After reading only the abstract, your audience should have a high-level technical understanding of how your proposal works.